.github/ISSUE_TEMPLATE/contribute.txt

```diff
@@ -0,0 +1,24 @@
+---
+name: Contribute
+about: Let us know you have an example you'd like to contribute
+
+---
+
+**Description**
+A clear and concise description of the concepts in your project and the problem your code solves. Please include any references you used. 
+
+**How to run the example**
+A short description of how to run your code, including an example. 
+
+**Link to your repo**
+Include a link to the repo you want to contribute. We will fork it. If we want to make any changes to the code we will create a pull request for you to approve.
```

Comment:
"If we want to make any changes to the code we will create a pull request for you to approve." Seems like something that we would have to be held to in the future...probably not what we want.

Perhaps simply suggest that we may make a pull request to their repo? "We may make a pull request to your repo if we would like your approval for certain code changes." Not sure about wording...will like to hear feedback from other reviewers.

Reply:
I'll add both options to the template